### PR TITLE
Fixed typo ‘Atribute’

### PR DESCRIPTION
**Pull request by jnm2**
That moment when you mistype something at source.dot.net 😱
image

### CODE REVIEW

File: src/System.Data.Common/src/Resources/Strings.resx [hunk 1]
```diff
@@ -109,7 +109,7 @@
   <data name="Xml_DataTableInferenceNotSupported" xml:space="preserve"><value>DataTable does not support schema inference from Xml.</value></data>
   <data name="Xml_MultipleParentRows" xml:space="preserve"><value>Cannot proceed with serializing DataTable '{0}'. It contains a DataRow which has multiple parent rows on the same Foreign Key.</value></data>
   <data name="Xml_IsDataSetAttributeMissingInSchema" xml:space="preserve"><value>IsDataSet attribute is missing in input Schema.</value></data>
-  <data name="Xml_TooManyIsDataSetAtributeInSchema" xml:space="preserve"><value>Cannot determine the DataSet Element. IsDataSet attribute exist more than once.</value></data>
+  <data name="Xml_TooManyIsDataSetAttributeInSchema" xml:space="preserve"><value>Cannot determine the DataSet Element. IsDataSet attribute exist more than once.</value></data>
```
Comment (jnm2) [inline src/System.Data.Common/src/Resources/Strings.resx]: Now I see that there is incorrect grammar in Strings.resx. Should I fix in this PR?

> IsDataSet attribute exist more than once.

It's either “attributes exist” or “attribute exists.”
Or, “More than one IsDataSet attribute exist” (verb agreement: “more ... exist”), which sounds more natural to me.

And it looks weird to capitalize ‘Element.’

❔

Comment (safern) [reply src/System.Data.Common/src/Resources/Strings.resx]: > More than one IsDataSet attributes exist

Agree, this one is more natural.

> And it looks weird to capitalize ‘Element.’

Yup, use lower case.

**Comment by safern** [comment]
> That moment when you mistype something at source.dot.net 😱

Nice find 😆

File: src/System.Data.Common/src/Resources/Strings.resx (hunk 1 above)
Comment (jnm2) [reply src/System.Data.Common/src/Resources/Strings.resx]: I doubted myself and edited it, but I just looked it up and I was right the first time:

> More than one IsDataSet attribute exists

Comment (safern) [reply src/System.Data.Common/src/Resources/Strings.resx]: Go for it.

**Comment by stephansmommy15** [comment]
Accident lol

**Comment by jnm2** [comment]
Copy/paste mistake resulting in weird message for the second commit, sorry.

**Pull request merged**